keyboards/chidori/keymaps/oled_sample/keymap.c

```diff
@@ -0,0 +1,225 @@
+/* Copyright 2019 ENDO Katsuhiro <ka2hiro@kagizaraya.jp>
+ * Copyright 2020 Masaya Uno
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#include "board.h"
+#include <stdio.h>
```

Comment:
I'm not sure that you need these. 
```suggestion
```

Reply:
`stdio.h` can be deleted. I used `snprintf()` in the first revision but deleted it.
But `board.h` is necessary because of the function `led_update_user()`.